.github/workflows/ruby.yml

```diff
@@ -20,14 +20,16 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Ruby
-    # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
-    # change this to (see https://github.com/ruby/setup-ruby#versioning):
-    # uses: ruby/setup-ruby@v1
-      uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: 2.7.1
-    - name: Install dependencies
-      run: bundle install
+    - name: Setup ENV
```

Comment:
This was an interesting one to get set up - a bit of knowledge of Linux commands go a long way!